test/token_whitelist/token_whitelist_suite_test.go

```diff
@@ -24,10 +24,9 @@ func init() {
 	)
 
 	TestRig.AddCoverageForContracts(
-		"../../build/internals/tokenWhitelistable/combined.json",
```

Comment:
@nostdm ^^ Other than this, i am happy, was this an intentional change ?

Reply:
Looks like @i-stam  changed it to be able to test the internal functions of the contract and increase the coverage.

Reply:
Basically, it is because I removed the bindings that were not actually needed, check this commit:
https://github.com/tokencard/contracts/pull/559/commits/76f4a267560b23819950b8fd42f9d7a2942ac592

the source of `tokenWhitelistable` is included in the exporter ;)